pkgs/games/mudlet/default.nix

```diff
@@ -18,24 +18,48 @@
 , pugixml
 , qtbase
 , qtmultimedia
+, discord-rpc
 , yajl
 }:
 
 let
-  luaEnv = lua.withPackages(ps: with ps; [
-    luazip luafilesystem lrexlib-pcre luasql-sqlite3 lua-yajl luautf8
+  overrideLua =
+    let
+      packageOverrides = self: super: {
+        # luasql-sqlite3 2.6.0-1 broke compatibility with lua 5.1.
```

Comment:
Is there an upstream issue which we could reference here?

Reply:
Yup! I filed one this morning: https://github.com/lunarmodules/luasql/issues/147

I updated the comment with a reference to that issue, and reworded slightly to clarify that it's not the upstream 2.6.0-1 release that broke compat. The problem is in the unreleased master branch.

Confusingly the `generated-packages.nix` entry for `luasql-mysqlite3` versions itself 2.6.0-1 but pins a commit that's newer than the release tag, so we require the override in this branch: https://github.com/NixOS/nixpkgs/blob/4d819f7910d2ad8ca7f2e87c0328d3c4541785bf/pkgs/development/lua-modules/generated-packages.nix#L2174-L2185